src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,86 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todoList: Todo[];
+  onSelect?: (todo: Todo) => void;
+  activeTodos: number | null;
```

Comment:
The `activeTodos` prop is used to toggle the visibility of an icon for a selected todo. Consider renaming it to `selectedTodoId` or something more descriptive to better convey its purpose and improve code readability.